spec/models/user_spec.rb

```diff
@@ -367,4 +366,21 @@
       it { expect(subject.admin?).to be_truthy }
     end
   end
+
+  describe '#full_name' do
+    subject { create(:user, first_name: 'first', last_name: 'last') }
+
+    it 'should compose #first_name' do
+      expect(subject.first_name).to eq('first')
+    end
+    it 'should compose #last_name' do
+      expect(subject.last_name).to eq('last')
+    end
+    it 'should return "first_name last_name"' do
+      expect(subject.full_name).to eq('first last')
+    end
+    it 'should belong to a valid User' do
+      expect(subject.valid?).to be true
+    end
```

Comment:
This is really testing whether our FactoryGirl factory is creating a valid User instance, but that is already covered by the test at line 12.